AutopanoPro.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+  <mime-type type="application/pano">
+    <sub-class-of type="text/plain" />
+    <comment xml:lang="en">Panorama project for AutopanoPro</comment>
+    <icon name="AutopanoPro" />
```

Comment:
it will never find that icon because it will never be exported.

Reply:
I guess I need to use `com.kolor.AutopanoPro` then instead?

Reply:
yes